CMakeLists.txt

```
@@ -339,6 +339,7 @@ endif()
 
 
 set(ROCM_LIBS "-L${ROCM_PATH}/lib -L${ROCM_PATH}/hip/lib -lamdhip64" CACHE STRING "Necessary libraries for ROCm")
+set(ACPP_FILESYSTEM_SEARCH_OPTIONS "Final Experimental Boost" CACHE STRING "Which filesystem libraries should be searched for")
```

Defining this as a single string seems to make the FindFilesystem module think that we're looking for a filesystem library called "Final Experimental Boost". Putting semicolons between the names makes CMake interpret it as a list. 

```suggestion
set(ACPP_FILESYSTEM_SEARCH_OPTIONS "Final;Experimental;Boost" CACHE STRING "Which filesystem libraries should be searched for")
```

---

thanks for checking! changing now